planner/core/exhaust_physical_plans.go

```diff
@@ -1546,9 +1546,23 @@ func (la *LogicalApply) exhaustPhysicalPlans(prop *property.PhysicalProperty) ([
 		return nil, true
 	}
 	join := la.GetHashJoin(prop)
+	var columns []*expression.Column
+	for _, colColumn := range la.CorCols {
+		columns = append(columns, &colColumn.Column)
+	}
+	count := getCardinality(columns, la.schema, la.stats) / la.stats.RowCount
```

Comment:
We should check `la.stats.RowCount == 0` especially, otherwise it would cause a panic.